setup.py

```diff
@@ -21,6 +21,7 @@
     install_requires=[
         'pyyaml',
         'requests',
+        'junit-xml'
```

Comment:
I think most users will not need this feature, similar to the `--word-list` option I added, can you put this under `extras_require` due to this?